### PR TITLE
Introduced split_clusters_during_rolling_update

### LINKED ISSUE #33: KUBE_PING requires option to separate clusters during Rolling Update
During the Rolling Update all Pods (new and old) are put in the same cluster. For some use cases (e.g. when an app deployed on WF is not backwards compatible and uses cache) we need to separate those two clusters. 
See also: https://issues.jboss.org/browse/CLOUD-2001

### PR DESCRIPTION
**Pull request by slaskawi**
Fixes #33

### CODE REVIEW

File: src/main/java/org/jgroups/protocols/kubernetes/Client.java [hunk 1]
```diff
@@ -92,22 +99,28 @@ protected String fetchFromKubernetes(String op, String namespace, String labels,
...
-                        Json phase=status.at("phase");
-                        if(phase != null && phase.isString() && !"Running".equals(phase.asString())) {
-                            log.trace("skipped pod with IP=%s as it is not running (%s)", podIP, phase);
-                            continue;
-                        }
-                    }
-                    if(!pods.contains(podIP))
-                        pods.add(podIP);
-                }
+            String parentDeployment = Optional.ofNullable(obj.at("metadata"))
```
Comment (slaskawi) [inline src/main/java/org/jgroups/protocols/kubernetes/Client.java]: @belaban Are you OK with this change?

File: src/main/java/org/jgroups/protocols/kubernetes/KUBE_PING.java [hunk 2]
```diff
@@ -206,6 +218,33 @@ public void findMembers(List<Address> members, boolean initial_discovery, Respon
                 list.stream().filter(phys_addr -> !cluster_members.contains(phys_addr)).forEach(cluster_members::add);
         }
 
+        if (split_clusters_during_rolling_update) {
+            if(physical_addr != null) {
+                String senderIp = ((IpAddress)physical_addr).getIpAddress().getHostAddress();
```
Comment (slaskawi) [inline src/main/java/org/jgroups/protocols/kubernetes/KUBE_PING.java]: @belaban Could you please have a look at this? Can I safely cast to `IpAddress`?

File: src/main/java/org/jgroups/protocols/kubernetes/Client.java (hunk 1 above)
Comment (belaban) [inline src/main/java/org/jgroups/protocols/kubernetes/Client.java]: Big change: I'd like to test this with minikube first... however the turnaround time for this is quite high .. stay tuned.

Comment (slaskawi) [inline src/main/java/org/jgroups/protocols/kubernetes/Client.java]: Ok, no problem. Take your time and keep me posted ;)

File: src/main/java/org/jgroups/protocols/kubernetes/KUBE_PING.java (hunk 2 above)
Comment (belaban) [reply src/main/java/org/jgroups/protocols/kubernetes/KUBE_PING.java]: No, this should be find as I don't have other impls of PhysicalAddress

**Pull request closed without merging**